oak_attestation_verification/src/verifier.rs

```diff
@@ -121,43 +122,166 @@ fn verify_dice_chain(evidence: &Evidence) -> anyhow::Result<()> {
 }
 
 fn verify_oak_restricted_kernel(
-    _evidence: &Evidence,
```

Comment:
Will this not include the actual evidence later to compare it with the endorsements and reference values?